src/integrationTest/java/uk/gov/hmcts/reform/sendletter/tasks/reports/DailyLetterUploadSummaryReportTest.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.sendletter.tasks.reports;
+
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.mail.javamail.JavaMailSender;
+
+import javax.mail.internet.MimeMessage;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+
+@SpringBootTest
+public class DailyLetterUploadSummaryReportTest {
```

Comment:
should we also include test for when recipient list is not present ?

Reply:
We have it from #396 as a unit test. Since these are cron based report tasks these integration tests purely stands for "kind of functional" ones, integrity verifiers.